__tests__/basic.js

```diff
@@ -144,3 +144,84 @@ test('thumbnail card with a long title and richCardWrapTitle set to default valu
 
   expect(base64PNG).toMatchImageSnapshot(imageSnapshotOptions);
 });
+
+test('absolute timestamp', async () => {
+  const activities = [
```

Comment:
The dump is pretty long, but seems no easy way to fix that. Feel free to leave it as-is.